container/tabs_test.go

```diff
@@ -19,3 +23,26 @@ func TestTabButton_Icon_Change(t *testing.T) {
 	b.Refresh()
 	assert.NotEqual(t, oldResource, icon.Resource)
 }
+
+func TestTab_ThemeChange(t *testing.T) {
+	a := test.NewApp()
+	defer test.NewApp()
+	a.Settings().SetTheme(theme.LightTheme())
+
+	tabs := NewAppTabs(
+		NewTabItem("a", widget.NewLabel("a")),
+		NewTabItem("b", widget.NewLabel("b")))
+	w := test.NewWindow(tabs)
+	w.Resize(fyne.NewSquareSize(100))
+
+	initial := w.Canvas().Capture()
+
+	a.Settings().SetTheme(theme.DarkTheme())
```

Comment:
This is really nitpicky since it's just test code, but I feel like we shouldn't be introducing *any* new code that calls deprecated functions (DarkTheme() and LightTheme()). Maybe we need a test theme package that has dark and light themes that aren't publicly exposed?

Reply:
I agree. Then we can turn deprecation warnings back on in staticcheck!